examples/post.janet

```diff
@@ -0,0 +1,21 @@
+(import janet-html)
+
+(def server-error-response
+  {:status 500
+   :body "Something went wrong, please try again later."})
+
+(defn create-response
+  [request]
+  (janet-html/encode
+    [:html
+     [:body
+      [:h1 "Post request example."]
+      (if request
+        [:p (string/format "Your request contained the following body: %s" (-> request (get :body) string))]
+        [:p "Send a POST request here to see its content in the response."])]]))
+
+(try
```

Comment:
Why wrap this in a `try`? I don't believe anything inside `create-response` can fail.